doc/pages/Manual.md

```diff
@@ -133,24 +347,25 @@ form: `name1+name2+name3:var` is the conjunction of `var` for each of `name1`,
 `name2` and `name3`, _i.e_ it is equivalent to `name1:var & name2:var &
 name3:var`
 
-#### Scopes
+##### Scopes
 
 The defined variables depend on the specific fields being defined. There are
 three scopes:
 
-1. Global variables correspond to the general current configuration (system
-   setup, OPAM configuration, current switch, etc.). For example `opam-version`,
-   `arch`, or `make`.
+1. Global variables correspond to the general current configuration, or to the
+   current switch settings (system setup, OPAM configuration, current switch
+   name, etc.). For example `opam-version`, `arch`, or `make`.
 2. Package variables have the form `package-name:var-name` and contain values
-   specific to a given package, e.g. `foo:installed` is a boolean variable that
-   can be used to check if package `foo` is installed, `foo:lib` is its library
-   installation directory. The prefix `package-name:` can be omitted within the
-   package's own OPAM file (when not shadowed by another variable), in which
-   case the variables will relate to the version of the package being defined.
-   Otherwise, the installed version is always used, and most variables will be
-   undefined when the package isn't installed.
+   specific to a given package, for example `foo:installed` is a boolean
+   variable that can be used to check if package `foo` is installed, and
+   `foo:lib` is its library installation directory. The prefix `package-name:`
+   can be replaced by `_:` within the package's own OPAM file, or simply omitted
+   when not shadowed by another variable. In this case, the variable always
+   relates to the version of the package being defined. Otherwise, `foo:bar`
```

Comment:
Hm, no, that was unclear: rest assured, if you are in the opam file defining some version of `p`, `p:var` will always refer to the variable at that version. `_:` doesn't do any magic besides selecting the package name; and indeed, it's new in 2.0.